### PR TITLE
Re-enable nf-core lint

### PR DESCRIPTION
**Pull request by matthdsm**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-cmgg/preprocessing/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,4 @@ results/
 testing/
 testing*
 *.pyc
-.nf-test*
```
Comment (nvnieuwk) [inline .gitignore:9]: This should be in the gitignore

**Review by nvnieuwk — APPROVED** [review]
One small comment

File: .gitignore (hunk 1 above)
Comment (matthdsm) [reply .gitignore:9]: `nf-core pipelines lint --fix files_unchanged` says not

Comment (nvnieuwk) [reply .gitignore:9]: Well it's wrong unless you want to commit your nf-test cache folder ;)

File: .gitignore [hunk 2]
```diff
@@ -6,4 +6,4 @@ results/
...
+null/
```
Comment (matthdsm) [inline .gitignore:9]:
```suggestion
null/
.nf-test*
```

**Pull request merged**